jwst/datamodels/schemas/resampmeta.schema.yaml

```diff
@@ -0,0 +1,27 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/resampmeta.schema"
+type: object
+  properties:
+    meta:
+      type: object
+      properties:
+        bunit_data:
```

Comment:
We shouldn't need to have BUNIT defined explicitly here, because `bunit.schema` is already inherited in both `image.schema` and `slitdata.schema`.

Furthermore, we could easily move the remaining keywords - NDRIZ, TEXPTIME, and RESWHT - into `core.schema`. They could be put into a new sub-section of `core.schema`, entitled "Resampling parameters". They'll only show up in products when they've got values.

So basically we don't need `resampmeta.schema` at all.

Reply:
That makes sense.  Just make sure they map to `meta.resample.product_exposure_time`, `meta.resample.weight_type`, etc.  I.e. they should have a subheading `resample`.